pkg/static/login.css

```diff
@@ -922,6 +929,112 @@ input[type=password] + .login-password-toggle .password-hide {
   display: none;
 }
 
+/* Dark */
+@media (prefers-color-scheme: dark) {
+  .login-pf .container,
+  .login-pf .details {
+    z-index: 2;
+  }
+
+  .login-pf .container {
+    background: #26292d;
+    color: #f0f0f0;
+  }
+
+  .login-pf .details {
+    background: #151515;
+    color: #f0f0f0;
+  }
+
+  .pf-c-button.pf-m-control,
+  .form-control[type=password],
+  .form-control[type=text] {
+    background: #393f44;
+    border-color: #393f44;
+    border-bottom-color: #6c6f72;
+    color: #f0f0f0;
+  }
+
+  a,
+  a:active,
+  a:focus,
+  a:hover {
+    color:#8ac0f6;
+  }
+
+  .pf-m-primary {
+    background-color: #06c;
+  }
+
+  .pf-m-primary:hover {
+    background-color: #004080;
+  }
+
+  .pf-m-tertiary {
+    background-color: transparent;
+    border-color:#8ac0f6;
+    color:#8ac0f6;
+  }
+
+  .pf-m-tertiary:active,
+  .pf-m-tertiary:focus,
+  .pf-m-tertiary:hover {
+    background-color: transparent;
+    border-color: #c5e0fb;
+    outline-color: #c5e0fb;
+    color: #c5e0fb;
+  }
+
+  .input-clear,
+  .inline .container .help-block {
+    color: #f0f0f0;
+  }
+
+  .host-line {
+    border-color: #34373b;
+  }
+
+  /* The × icon's focus ring */
+  .host-remove:focus {
+    outline-color: #f1f1f1;
+  }
+
+  /* The × icon */
+  .host-remove::before {
+    background-color: #f1f1f1;
+  }
+
+  .unsupported-browser ul {
+    color: #999;
+  }
+
+  .pf-c-alert.pf-m-inline.pf-m-danger {
+    background: #1E2125;
+    border-color: #f64747;
+  }
+
+  .pf-c-alert.pf-m-danger > svg,
+  .pf-c-alert.pf-m-danger .pf-c-alert__title {
+    color: #f64747;
```

Comment:
It seems that patternfly uses a slightly different color for this one:

```
[jelle@t14s][~/projects/patternfly]%git grep pf-color-red-9999;
src/patternfly/base/themes/dark/_variables.scss:  --pf-global--palette--red-9999: #{$pf-color-red-9999};
src/patternfly/sass-utilities/themes/dark/colors.scss:$pf-color-red-9999: #fe5142;
src/patternfly/sass-utilities/themes/dark/scss-variables.scss:$pf-global--danger-color--100: $pf-color-red-9999;
```

It is more orange'ish somehow :)

Reply:
I pulled the colors directly from their mockups. They might've changed it?